query/security/ldap/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -15,14 +15,19 @@
 
     <service id="ldapFieldProvider" interface="org.codice.ddf.admin.api.FieldProvider">
         <bean class="org.codice.ddf.admin.ldap.LdapFieldProvider">
-            <argument ref="configuratorFactory" />
+            <argument ref="configuratorFactory"/>
             <argument ref="featureActions"/>
             <argument ref="managedServiceActions"/>
             <argument ref="propertyActions"/>
             <argument ref="serviceActions"/>
+            <property name="installEmbeddedLdap" ref="installEmbeddedLdap"/>
         </bean>
     </service>
 
+    <reference id="installEmbeddedLdap"
+               interface="org.codice.ddf.admin.api.fields.FunctionField"
+               availability="mandatory" filter="(id=installEmbeddedLdap)"/>
```

Comment:
We'd probably want to change the ref id and the property name to something more generic like 'ldapFields', but that's a nice idea. 👍